packages/ui-components/src/components/ComboBox/ComboBox.stories.tsx

```diff
@@ -27,26 +29,24 @@ export default {
     },
   },
   decorators: [
-    (Story) => (
+    (story: StoryFunction) => (
       <div className="jn-pb-12">
-        <PortalProvider>
-          <Story />
-        </PortalProvider>
+        <PortalProvider>{story()}</PortalProvider>
       </div>
     ),
   ],
 }
 
-const Template = ({ children, ...args }) => {
+const Template = ({ children, ...args }: TemplateProps) => {
   return <ComboBox {...args}>{children}</ComboBox>
 }
 
 // define prop types fro Template
-Template.propTypes = {
-  children: PropTypes.any,
-}
+type TemplateProps = {
+  children: React.ReactNode
+} & ComboBoxProps
```

Comment:
I just learned this https://github.com/microsoft/TypeScript/wiki/Performance#preferring-interfaces-over-intersections
So we should use interfaces over types in a case of intersection.